File: search_a_2d_matrix/search_2d_matrix.cpp
class Solution {
public:
    bool searchMatrix(vector<vector<int>>& matrix, int target) {
        vector<int>newl;
        for(int i=0;i<matrix.size();i++){
            for(int j=0;j<matrix[i].size();j++){
                newl.push_back(matrix[i][j]);
            }
        }
        for(int i=0;i<newl.size();i++){
            if(newl[i]==target){
                return true;
            }
        }
        return false;
    }
};

File: 74_search_a_2d_matrix/search_2d_matrix.py
class Solution:
    def searchMatrix(self, matrix: List[List[int]], target: int) -> bool:
        newl=[]
        for i in range(0,len(matrix)):
            for j in range(0,len(matrix[i])):
                newl.append(matrix[i][j])
        
        if (target in newl):
            return True
        else:
            return False